friends/src/Components/UpdateForm.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+
+
+export default class UpdateForm extends React.Component{
+    constructor(props){
+        super(props)
+
+        this.state = {
+            person:{
+                name:'',
+                age:'',
+                email:''
+            }
+        }
+    }
+
+
+    handleChange = (e) =>{
+        e.persist()
+        let value = e.target.value
+        this.setState(prevState =>({
+            person:{
+                ...prevState.person,
+                [e.target.name]:value
+            }
+        }))
+    }
+
+    updatePerson = (e) =>{
+
+        e.preventDefault();
+
+        this.props.updateFriend(this.props.item.id, this.state.person)
+    }
+
+    render(){
+        return(
```

Comment:
There is no way to navigate back to the home or list page from the update screen. Fix this by adding navmenu or re-routing to the list page.